client/state/analytics/actions/record.js

```diff
@@ -18,8 +19,14 @@ export const recordEvent = ( service, args ) => ( {
 export const recordGoogleEvent = ( category, action, label, value ) =>
 	recordEvent( 'ga', { category, action, label, value } );
 
-export const recordTracksEvent = ( name, properties ) =>
-	recordEvent( 'tracks', { name, properties } );
+export const recordTracksEvent = ( name, properties ) => {
+	//Each event is fed a device type property allowing the device focussed analysis of events
+	const finalProperties = {
+		...properties,
+		device_type: resolveDeviceTypeByViewPort(),
+	};
+	return recordEvent( 'tracks', { name, properties: finalProperties } );
```

Comment:
This is huge and will affect all the events. Thus, even though it should work for what we want, I'd like us to not rush it but hear from the data team first. @aaronyan @withinboredom could you pitch in here? 

@jdc91 In the future, I'd recommend to isolate this kind of change into a separate PR so we can have a clearer view to isolated pieces. In this case, this change together with the viewport package update can be extracted from the experiment itself. Of course, let's not do that now, just wanted to let you be aware of this strategy :)

Reply:
Thanks @southp I will keep that in mind.

Reply:
If you think this is a bit too big of a change, I can go over the exact event that needs this prop and introduce it. Emre has provided me with a set of fitting events to include so this will be a less impacting change.

Reply:
Moved this to each sepcific event.